OutOfSchool/OutOfSchool.IdentityServer/Data/Migrations/OutOfSchoolMigrations/20211107202949_InitialMigration.cs

```diff
@@ -474,23 +573,23 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         column: x => x.ProviderId,
                         principalTable: "Providers",
                         principalColumn: "Id");
-                });
+                })
+                .Annotation("MySql:CharSet", "utf8mb4");
 
             migrationBuilder.CreateTable(
                 name: "Applications",
                 columns: table => new
                 {
-                    Id = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
+                    Id = table.Column<byte[]>(type: "binary(16)", nullable: false),
                     Status = table.Column<int>(type: "int", nullable: false, defaultValue: 1),
-                    CreationTime = table.Column<DateTimeOffset>(type: "datetimeoffset", nullable: false),
-                    WorkshopId = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
-                    ChildId = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
-                    ParentId = table.Column<Guid>(type: "uniqueidentifier", nullable: false)
+                    CreationTime = table.Column<DateTimeOffset>(type: "datetime(6)", nullable: false),
```

Comment:
one other thing that came to mind, can you check if we have time zone information (as we use `DateTimeOffset` in code) in database or that there is a correct conversion while writing/reading (e.g, we save `10:00+02:00` and it is stored as `08:00Z` and not as `10:00Z`)

Reply:
yes, it converts time as UTC.